test/test_wsgi.py

```diff
@@ -12,8 +12,6 @@ def tflow():
 
 class TestApp:
 
-    def __init__(self):
-        self.called = False
```

Comment:
py.test shouldn't collect TestApp at all, as TestApp does not contain tests but is an example app used for testing. Thanks for catching that. Can you please just rename TestApp to ExampleApp or DummyApp and re-add the constructor?